### PR TITLE
Notification

### PR DESCRIPTION
**Pull request by VictorAny**
- Fixes issues with server endpoint.
- Implements api calls in App Engine Manager
- Saves user profile into nsuserdefaults 
- Other important and cool stuff I can't remember.

### CODE REVIEW

File: PiRemote/PiRemote/MainUser.swift [hunk 1]
```diff
@@ -5,25 +5,60 @@
...
+    
     // Remot3.it
     var email : String?
-    var apiKey: String?
-    var token: String?
+    
+    // Weird. Apparently this is the global API key and they specified they planned on having individual keys in the future..but I didn't find anything
+    // in the docs
+    // see : http://docs.weaved.com/v21/discuss/577bde161cf3cb0e0048e357
+    var apiKey: String = "WeavedDemoKey$2015"
```
Comment (mmrtnz) [inline PiRemote/PiRemote/MainUser.swift:23]: I couldn't find anything either. Have a hunch that it's not gonna update anytime soon 😷

Comment (VictorAny) [reply PiRemote/PiRemote/MainUser.swift:23]: Lol I know right...They aren't really keeping up with this stuff well

File: AppEngineServer/server/main.py [hunk 2]
```diff
@@ -15,87 +15,130 @@
...
-		except:
-			self.writeErrorResponse("Account data not fully provided.")
+		data = json_req
+		print data
+		print "Entering account handler!"
+		service_ids = data["service_ids"]
+		for id_ in service_ids:
+			# Checks if account exists at all..
+			# Means only there cannot be two accounts with the same service_id
+			accountExists = self.validateAccount(id_)
```
Comment (mmrtnz) [inline AppEngineServer/server/main.py:101]: Perfect ⭐️

**Comment by mmrtnz** [comment]
Awesome job!

**Pull request merged**